test/runtime/test-adapter-clone.js

```diff
@@ -0,0 +1,10 @@
+var TestAdapter = require('./test-adapter');
```

Comment:
It actually needs to be in a separate file since it is lazily loaded using a deferred require.